### PR TITLE
moved __main__ code in __main__.py

### PR DESCRIPTION
**Pull request by formula-spectre**
so I tried to make this work; and it actually does surprisingly. as I get this output:
```
. .venv/bin/activate
python -m keyring_pass
testing with svc='test' user='asd' pwd='zxc'
OK: get_password(): matches
OK: get_credential(): found
OK: get_credential(): username matches
OK: get_credential(): password matches
```
to reproduce this before merging just download my fork, activate a venv, then `pip install . && python -m keyring_pass`

### CODE REVIEW

File: keyring_pass/__main__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from . import * # idk if this is bad practice, but as a quick fix it works _shrug_
```
Comment (nazarewk) [inline keyring_pass/__main__.py]: from what I'm seeing this should be enough:
```suggestion
import sys

import keyring

from keyring_pass import PasswordStoreBackend
```

File: keyring_pass/__main__.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+if __name__ == "__main__":
```
Comment (nazarewk) [inline keyring_pass/__main__.py]: you could remove the `if` clause and dedent rest of the file

File: keyring_pass/__main__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            errors.append(
+                f"get_credential(): {credential.password=} doesn't match {pwd=}"
+            )
+        else:
+            print(f"OK: get_credential(): password matches")
+
+    for err in errors:
+        print(f"ERROR: {err}")
+    if errors:
+        sys.exit(1)
```
Comment (nazarewk) [inline keyring_pass/__main__.py]:
```suggestion
svc = "test"
user = "asd"
pwd = "zxc"

keyring.set_keyring(PasswordStoreBackend())
errors = []
print(f"testing with {svc=} {user=} {pwd=}")

try:
    keyring.set_password(svc, user, pwd)
    returned = keyring.get_password(svc, user)
    credential = keyring.get_credential(svc, None)
finally:
    keyring.delete_password(svc, user)

if returned != pwd:
    errors.append(f"get_password(): {returned=} != {pwd=}")
else:
    print(f"OK: get_password(): matches")

if not credential:
    errors.append(f"get_credential() not found ({credential=})")
else:
    print(f"OK: get_credential(): found")
    if credential.username != user:
        errors.append(
            f"get_credential(): {credential.username=} doesn't match {user=}"
        )
    else:
        print(f"OK: get_credential(): username matches")

    if credential.password != pwd:
        errors.append(
            f"get_credential(): {credential.password=} doesn't match {pwd=}"
        )
    else:
        print(f"OK: get_credential(): password matches")

for err in errors:
    print(f"ERROR: {err}")
if errors:
    sys.exit(1)
```

**Comment by nazarewk** [comment]
added some minor fixes, but otherwise it really is exactly as simple as it looks :)

**Comment by formula-spectre** [comment]
lgtm too

**Pull request merged**